gestalt-inject-java/src/main/java/org/terasology/gestalt/annotation/processing/ResourceProcessor.java

```diff
@@ -20,6 +20,9 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+/**
+ * Annotation processor to create resources for types.
```

Comment:
```suggestion
 * Annotation processor to list resources for types.
```

This processor does not create the resources, it just lists the resources that are there. We need to do this because listing resources packed in an APK on Android is prohibitively expensive.